hackathon_site/hardware/serializers.py

```diff
@@ -211,12 +212,21 @@ def validate(self, data):
         return data
 
     def create(self, validated_data):
+
+        # make a copy of validated_data
+        validated_data_copy = validated_data
```

Comment:
might be better to use validated_data.copy() to ensure it's an actual copy of the variable